lblod/frontend-gelinkt-notuleren
# GN-4360: Unable to edit meeting after reverting signatures + no alert

### Overview


#### Connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4360


### Setup
Usual setup, nothing special

### How to test/reproduce
You need to have a Notulen ready to be signed.

1. Sign with one user
2. Observer that you cannot edit the meeting anymore
3. Sign with second user
4. Remove the second user signature
5. Observe the notification
6. Observer that you cannot edit the meeting still
7. Remove the first signature
8. Observe there's no more notification
9. Observer that you now _can edit the meetinf_

See video at - https://share.cleanshot.com/JJVHDGlj



### Checks PR readiness
